### ISSUE TITLE
Syntax error: Selector ":root" is not pure (pure selectors must contain at least one local class or id)

### ISSUE THREAD

**Issue by imtheaman** [henripar/react-lite-month-picker#3]
I'm getting this error while using this pkg in next.js

Error:
<img width="964" alt="Screenshot 2024-03-11 at 8 16 22 PM" src="https://github.com/henripar/react-lite-month-picker/assets/62852336/d244bb63-0e09-4421-94d1-5992b7e210b4">

```
./node_modules/react-lite-month-picker/src/components/MonthInput/MonthInput.module.css:21:1
Syntax error: Selector ":root" is not pure (pure selectors must contain at least one local class or id)

  19 | }
  20 | 
> 21 | :root {
     | ^
  22 |   --input-bg-color: #fff;
  23 |   --input-hover-bg-color: #fff;
```

**Comment by henripar** [comment]
Thanks! 

This is now fixed in version 1.2.18

**Comment by imtheaman** [comment]
Thanks! @henripar, I really gotta use it in my current project. It's gonna save me a lot of time
